src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,29 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime()
+        if(!image) {
+            return
+        }
+        try {
+            retry(3) {
+                def dd = e.git.url
+                echo "YZAINEE: ${dd}"
+                def response = bayesianAnalysis url: 'https://bayesian-link', gitUrl: e.git.url, ecosystem: image
```

Comment:
the conventions is to use `()` when assigning. So it must be 

`def response = bayesianAnalysis(url: '....', gitUrl: e.git.url, ecosystem: image)`